frontend/src/libs/schema.js

```diff
@@ -0,0 +1,110 @@
+export const instructorSchema = {
```

Comment:
I thought I moved all the schemas to a typescript file?

Reply:
removed outdated schema.js